mods/cnc/mod.yaml

```diff
@@ -29,13 +18,22 @@ Packages:
 	temperat.mix
 	winter.mix
 	desert.mix
-	snow.mix
 	~movies-gdi.mix
 	~movies-nod.mix
 	~movies.mix
 	~scores.mix
 	~scores2.mix
 	~transit.mix
+	./mods/cnc/bits/snow.mix
```

Comment:
That's the intention. The prefix is required otherwise the game cannot find it. See https://github.com/OpenRA/OpenRA/pull/10554#discussion_r51493376